src/static/js/event_new.js

```diff
@@ -0,0 +1,183 @@
+                '</div>',
+                html.content_footer()
+            ].join("\n");
+        },
+
+        bind: function(){
+            $("#reset").button().click(function(){
+                event_new.reset();
+            });
+            $("#add").button().click(function(){
+                event_new.create_and_edit = false;
+                $("#asm-content button").button("disable");
+                check_for_similar();
+            });
+            $("#addedit").button().click(function(){
+                event_new.create_and_edit = true;
+                $("#asm-content button").button("disable");
+                check_for_similar();
+            });
+            //insert values to corresponding fields when a location is selected
+            $("#location").personchooser().bind("personchooserchange", function(event, rec){
+                $("#address").val(html.decode(rec.OWNERADDRESS));
+                $("#town").val(html.decode(rec.OWNERTOWN));
+                $("#county").val(html.decode(rec.OWNERCOUNTY));
+                $("#postcode").val(html.decode(rec.OWNERPOSTCODE));
+                $("#country").val(html.decode(rec.OWNERCOUNTRY));
+            });
+
+            //insert value to the same date of the chosen Start date if End date field is empty
+            $("#startdate").bind("change", function(){
+                if($("#enddate").val() == "" && $("#startdate").val() != "")
+                    $("#enddate").val($("#startdate").val());
+            });
+
+            const check_for_similar = async function(){
+                if(!event_new.validation()){
+                    $("#asm-content button").button("enable");
+                    return;
+                }
+//                let formdata = "mode=similar&" + $("#emailaddress, #mobiletelephone, #surname, #forenames, #address").toPOST();
+                add_event();
+            };
+
+            const add_event = async function(){
+                if(!event_new.validation())
+                {
+                    $("#asm-content button").button("enable");
+                    return;
+                }
+                header.show_loading(_("Creating..."));
+                try{
+                    let formdata = "ownerid=" + $("#location").personchooser().val() + "&" + $("input, textarea, select, #location").toPOST();
+                    let eventid = await common.ajax_post("event_new", formdata);
+                    if(eventid && event_new.create_and_edit)
+                        common.route("event?id=" + eventid);
+                    else
+                        header.show_info(_("Event successfully created"));
+                }
+                finally{
+                    $("#asm-content button").button("enable");
+                }
+            };
+
+            // CONFIG
+            $("#countryrow").toggle( !config.bool("HideCountry") );
+            $("#statecounty").toggle( !config.bool("HideTownCounty") );
+
```

Comment:
Country isn't enabled by default, and should be hidden by config option here

Reply:
@bobintetley it seems like the code does exactly that, no? the country is hidden by default (or by the config option setting)

Reply:
It does, clearly I'm blind!